orangecontrib/text/widgets/owtwitter.py

```diff
@@ -0,0 +1,275 @@
+import functools
+
+from PyQt4 import QtGui, QtCore
+from PyQt4.QtGui import QComboBox, QDialog
+
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.widget import OWWidget
+
+from orangecontrib.text import twitter
+from orangecontrib.text.corpus import Corpus
+from orangecontrib.text.widgets.ownyt import _i
+from orangecontrib.text.languge_codes import lang2code
+
+
+class APICredentialsDialog(QDialog):
+    size_hint = (470, 100)
+    widgets_min_width = 380
+
+    def __init__(self, parent, windowTitle="Twitter API credentials"):
```

Comment:
`credentials` should start with a camel-case according to our latest [human interface guidelines](https://github.com/biolab/orange3/blob/master/CONTRIBUTING.md#human-interface-guidelines).